driver-mapping/src/test/java/com/datastax/driver/mapping/MapperInvalidAnnotationsTest.java

```diff
@@ -261,27 +265,4 @@ public void setNotReadable(int i) {
 
     }
 
-    @Test(groups = "unit", expectedExceptions = IllegalArgumentException.class,
```

Comment:
You are right, these exceptions have became log warnings, see [here](https://github.com/datastax/java-driver/blob/4eaec3f16bb07acf3bec06d9bd51952dc436e35f/driver-mapping/src/main/java/com/datastax/driver/mapping/DefaultPropertyMapper.java#L237-L244). Not sure though if we should throw an exception or just log a warning after all.